docs/tutorials/how-to-estimator.ipynb

```diff
@@ -0,0 +1,174 @@
+    ") as estimator:\n",
+    "    theta1 = [0, 1, 1, 2, 3, 5]\n",
+    "    theta2 = [0, 1, 1, 2, 3, 5, 8, 13]\n",
+    "    theta3 = [1, 2, 3, 4, 5, 6]\n",
+    "\n",
+    "    # calculate [ <psi1(theta1)|H1|psi1(theta1)> ]\n",
+    "    psi1_H1_result = estimator(circuit_indices=[0], observable_indices=[0], parameter_values=[theta1])\n",
+    "    print(psi1_H1_result)\n",
+    "\n",
+    "    # calculate [ <psi1(theta1)|H2|psi1(theta1)>, <psi1(theta1)|H3|psi1(theta1)> ]\n",
+    "    psi1_H23_result = estimator(circuit_indices=[0, 0], observable_indices=[1, 2], parameter_values=[theta1]*2)\n",
+    "    print(psi1_H23_result)\n",
+    "\n",
+    "    # calculate [ <psi2(theta2)|H2|psi2(theta2)> ]\n",
+    "    # Note that you don't need to specify the labels \"circuit_indices\", \"observable_indices\", or \"parameter_values\", as long as they are specified in that order.\n",
+    "    psi2_H2_result = estimator([1], [1], [theta2])\n",
+    "    print(psi2_H2_result)\n",
+    "\n",
+    "    # calculate [ <psi1(theta1)|H1|psi1(theta1)>, <psi1(theta3)|H1|psi1(theta3)> ]\n",
+    "    psi1_H1_result2 = estimator([0, 0], [0, 0], [theta1, theta3])\n",
+    "    print(psi1_H1_result2)\n",
+    "\n",
+    "    # calculate [ <psi1(theta1)|H1|psi1(theta1)>,\n",
+    "    #             <psi2(theta2)|H2|psi2(theta2)>,\n",
+    "    #             <psi1(theta3)|H3|psi1(theta3)> ]\n",
+    "    psi12_H23_result = estimator([0, 1, 0], [0, 1, 2], [theta1, theta2, theta3])\n",
+    "    print(psi12_H23_result)\n",
+    "   ```\n",
+    "{: codeblock}\n",
+    "\n",
+    "Output:\n",
+    "```text\n",
+    "EstimatorResult(values=array([1.55273438]), metadata=[{'variance': 8.897655487060547, 'shots': 1024}])\n",
+    "EstimatorResult(values=array([-0.55664062, 0.07421875]), metadata=[{'variance': 0.6901512145996094, 'shots': 1024}, {'variance': 1.9938812255859375, 'shots': 1024}])\n",
+    "EstimatorResult(values=array([0.19921875]), metadata=[{'variance': 0.9603118896484375, 'shots': 1024}])\n",
+    "EstimatorResult(values=array([1.515625 , 0.97460938]), metadata=[{'variance': 9.603851318359375, 'shots': 1024}, {'variance': 11.997127532958984, 'shots': 1024}])\n",
+    "EstimatorResult(values=array([ 1.55078125, 0.13085938, -1.04101562]), metadata=[{'variance': 9.120590209960938, 'shots': 1024}, {'variance': 0.9828758239746094, 'shots': 1024}, {'variance': 1.2807121276855469, 'shots': 1024}])\n",
+    "```\n",
+    "\n",
+    "## Next steps\n",
+    "{: #next-steps}\n",
+    "\n",
+    "- Use the [Getting started guide](/docs/quantum-computing?topic=quantum-computing-quickstart) to create an instance and run your first job.\n",
+    "- Review [Get started with the Sampler primitive](/docs/quantum-computing?topic=quantum-computing-example-estimator) for a step-by-step example of using this primitive.\n",
+    "- Use Qiskit [tutorials](https://qiskit.org/documentation/tutorials.html){: external} to learn how to create circuits with Qiskit.\n"
```

Comment:
```suggestion
    "## Next steps\n",
    "{: #next-steps}\n",
    "\n",
    "- Use the [Getting started guide](/docs/quantum-computing?topic=quantum-computing-quickstart) to create an instance and run your first job.\n",
    "- Review [Get started with the Sampler primitive](/docs/quantum-computing?topic=quantum-computing-example-estimator) for a step-by-step example of using this primitive.\n",
    "- Use Qiskit [tutorials](https://qiskit.org/documentation/tutorials.html){: external} to learn how to create circuits with Qiskit.\n"
```
I would suggest we just remove this section for now and add one when we have other pages.